src/pages/home-page/hero-image/hero-image.module.scss

```diff
@@ -70,44 +75,75 @@
     .content {
         max-width: 1400px;
     }
+
+    .small-title {
+        font-size: 42px;
+    }
+
+    .big-title {
+        font-size: 85px;
+    }
+
+    .overlay-button {
+        height: 47px;
+    }
 }
 
 @media (980px <=width < 1400px) {
```

Comment:
that is a very strange media query
and something should be the default

Reply:
Same, it's still in process.